phpunit.xml.dist

```diff
@@ -12,7 +12,10 @@
         <var name="driver_config_factory" value="Behat\Mink\Tests\Driver\Selenium2Config::getInstance" />
 
         <!--server name="WEB_FIXTURES_HOST" value="http://test.mink.dev" /-->
-        <!--server name="WEB_FIXTURES_BROWSER" value="firefox" /-->
+        <!-- MacOS -->
+        <!--<server name="WEB_FIXTURES_HOST" value="http://docker.for.mac.localhost:8002"/>-->
```

Comment:
I'm not sure about this line and previous line. Docker isn't Mac-only solution and Java approach suggested in updated testing instructions (in same PR) also works on all OSes.

What does `docker.for.mac.localhost` mean? Is this some magic hostname, that Docker creates?

Reply:
This is a special host which docker is exposing to a container which allow container to connect to host

https://docs.docker.com/v17.09/docker-for-mac/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host